dashboard/models.py

```diff
@@ -159,18 +153,31 @@ def get_supported_courses(self):
 
 
 class Course(models.Model):
-    id = models.BigIntegerField(primary_key=True, verbose_name="Unizin Course Id", db_column='id', editable=False)
+    id = models.BigIntegerField(primary_key=True, verbose_name="Unizin Course Id", db_column='id')
     canvas_id = models.BigIntegerField(verbose_name="Canvas Course Id", db_column='canvas_id')
-    term_id = models.ForeignKey(AcademicTerms, verbose_name="Term Id", on_delete=models.SET_NULL, db_column='term_id', null=True, db_constraint=False)
+    term = models.ForeignKey(AcademicTerms, verbose_name="Term Id", on_delete=models.SET_NULL, db_column="term_id", null=True, db_constraint=False)
```

Comment:
Can the "verbose_name" field also be changed to "Term", instead of "Term Id"?

Reply:
Sure! See latest commit.